modules/ROOT/pages/prereq-aws-terraform.adoc

```diff
@@ -96,8 +97,12 @@ You can also include the following set of optional environment variables:
 docker load -i pcp-1.0.tar.gz
 ----
 
-. Record the image ID after running the `docker load` command. This is required in the following step.
-
+. Record the image ID after running the `docker load` command. Run the following command to retrieve the image ID, which is required for the next step:
```

Comment:
No need to run gravity images here. The image ID you need to use in the next step shows up right after you run docker load in the previous step

Reply:
Updated this - thank you!